src/resources/SecurityCache/SecurityCache.ts

```diff
@@ -71,12 +72,30 @@ export default class SecurityCache extends Ressource {
         return this.api.get<SecurityProviderModelWithStatus[]>(SecurityCache.providersUrl);
     }
 
-    providerSchedules() {
```

Comment:
This is the breaking change. I'm assuming it's fairly safe to remove because it doesn't work unless you had a provider with the ID "schedules". In that scenario, the response type would be invalid.

Reply:
👍  I don't know what happened here, I've juste checked and it isn't used in our internal stuff.